lib/Sema/SourceLoader.cpp

```diff
@@ -34,13 +34,20 @@ using namespace swift;
 // FIXME: Basically the same as SerializedModuleLoader.
 using FileOrError = llvm::ErrorOr<std::unique_ptr<llvm::MemoryBuffer>>;
 
-static FileOrError findModule(ASTContext &ctx, StringRef moduleID,
+static FileOrError findModule(ASTContext &ctx, Identifier moduleID,
                               SourceLoc importLoc) {
   llvm::SmallString<128> inputFilename;
```

Comment:
When is the source loader used? Should we test this?

Reply:
done